File: shoebill_libs/LtrpApi/src/main/java/lt/ltrp/event/player/PlayerEvent.java
package lt.ltrp.event.player;


import lt.ltrp.object.LtrpPlayer;

/**
 * @author Bebras
 *         2016.04.07.
 */
public class PlayerEvent extends net.gtaun.shoebill.event.player.PlayerEvent {

    public PlayerEvent(LtrpPlayer player) {
        super(player);
    }

    @Override
    public LtrpPlayer getPlayer() {
        return (LtrpPlayer)super.getPlayer();
    }
}

File: shoebill_plugins/PlayerPlugin/src/main/java/lt/ltrp/player/event/PlayerEvent.kt
package lt.ltrp.player.event


import lt.ltrp.player.`object`.LtrpPlayer
import net.gtaun.util.event.Event

/**
 * @author Bebras
 *         2016.04.07.
 *         Base event related to [LtrpPlayer]
 */
abstract class PlayerEvent(open val player: LtrpPlayer) : Event() {

}
